testdata/baselines/reference/submodule/conformance/typeFromJSInitializer.errors.txt

```diff
@@ -37,6 +36,8 @@ a.js(41,12): error TS2345: Argument of type '"ok"' is not assignable to paramete
     
     // should get any on parameter initialisers

```

Comment:
I don't understand what is going on here--the differing errors appear to match the comments better than before, so maybe the old baselines are wrong?